src/main/java/mate/academy/exception/RegisterException.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.exception;
+
+public class RegisterException extends RuntimeException {
+
+    public RegisterException(String message) {
+        super(message);
+    }
+
+    public RegisterException(String message, Throwable cause) {
+        super(message, cause);
+    }
+}
```

Comment:
This exception unnecessary, for register you should use AuthenticationException.
```suggestion

```